### ISSUE TITLE
Asm misses some of the instructions

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/compiler-explorer#362]
e.g. ROL is missing (though I see it in Felix's source). I think there's an issue with the docenizer parsing the combined pages.

Also, ideally the docenizer should output everything in sorted order to minimize the git diffs when only a few things change.

**Comment by mattgodbolt** [comment]
Similarly all the vector instructions aren't being picked up, probably as they have suffixes (like `vbroadcastss` etc : https://godbolt.org/g/ftSycN

**Comment by RabsRincon** [comment]
Would a solution like yours on prefixes work?

**Comment by mattgodbolt** [comment]
I don't think so. There are many suffixes and prefixes and they're noted in the x86 asm reference so we should try and parse them out and use them as aliases of possible

**Comment by mattgodbolt** [comment]
Missing `cmpltpd` (from https://godbolt.org/g/M9aRB4)

**Comment by mattgodbolt** [comment]
also `seto` https://godbolt.org/g/TsoS5B

**Comment by mattgodbolt** [comment]
https://github.com/StanfordPL/x64asm/blob/develop/src/x86.csv might be a better solution

**Comment by mattgodbolt** [comment]
Noticed it's missing `cltq` the other day

**Comment by mattgodbolt** [comment]
See also #1072

**Comment by phuclv90** [comment]
shr and movbe was mentioned in #695
also #362

**Comment by mattgodbolt** [comment]
Whoops; the `/g/` links here are mostly broken as they point to xania.org (my old "staging" setup). Will update with fixed links (or visit goo.gl/BLAH and fix up the url)
